R/utils_general.R

```diff
@@ -23,3 +23,44 @@ check_if_packages_are_available <- function(packages_ch) {
     )
   }
 }
+
+#' Access field Pandora help tooltip
+#'
+#' This functions allows you to call the 'tooltip' help/documentation message
+#' present on the Pandora webpage for a given entry.
+#'
+#' @param tab character. Name of the table of the field of interest
+#' @param field character vector. Name of the field of interest
+#' @param con database connection object
+#'
+#' @return help text of the given field
+#' 
+#' @name field_help
+#' 
+#' @export
+
+field_help <- function(tab, field, con) {
+  
+  if ( !tab %in% sidora.core::pandora_tables ) {
+    stop(crayon::red("[sidora.core] error: "), "Select one valid PANDORA SQL table.")
+    
+  }
+  
+  if (!field %in% sidora.core::pandora_column_types$col_name ) {
+    stop(crayon::red("[sidora.core] error: "), "Field name is not recognised.")
+  }
+  
+  pandora_help <- sidora.core::get_df("TAB_Field_Comment", con) %>% tibble::as_tibble()
```

Comment:
Why the `%>% tibble::as_tibble()`? Doesn't  `get_df` already yield a tibble? :thinking:

Reply:
Left over from when I was running `get_con`